Dockerfile

```diff
@@ -54,6 +54,7 @@ RUN if test "${WITH_ORACLE}" = "ON"; then \
       -DWITH_POINT_Z_M=1 \
       -DWITH_CAIRO=1 \
       -DWITH_RSVG=1 \
+      -DUSE_PROJ=1 \
```

Comment:
The only usage of `USE_PROJ` I see is to set `USE_TILE_API`:
https://github.com/MapServer/MapServer/blob/85e294e8c8ce3c9c455fea0137d4af1ee2feced0/maptile.h#L33-L35
Then shouldn't we directly use: `-DUSE_TILE_API=1` for me it looks clearer...

Reply:
Thank you @sbrunner for reviewing the pull-request. 

The option `-DUSE_PROJ=1` seems to be the only way set `USE_TILE_API`. I have tested `-DUSE_TILE_API=1` but it does not work (`CMake Warning Manually-specified variables were not used by the project: USE_TILE_API`) because it is not a configurable option for cmake:  https://github.com/MapServer/MapServer/blob/85e294e8c8ce3c9c455fea0137d4af1ee2feced0/mapserver-config.h.in#L4

Reply:
OK, thanks :-)